web/components/feed/activity-items.ts

```diff
@@ -362,7 +337,9 @@ function getCommentThreads(
   comments: Comment[],
   contract: Contract
 ) {
-  const parentComments = comments.filter((comment) => !comment.replyToCommentId)
+  let parentComments = comments.filter((comment) => !comment.replyToCommentId)
+  if (contract.outcomeType === 'FREE_RESPONSE')
+    parentComments = comments.filter((comment) => !comment.replyToCommentId)
```

Comment:
Not sure I understand — this is the same filter operation as above?

Reply:
Whoops! not sure what that was